lms/envs/common.py

```diff
@@ -1623,17 +1630,49 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 ################################# WIKI ###################################
 from lms.djangoapps.course_wiki import settings as course_wiki_settings  # pylint: disable=wrong-import-position
 
+# .. toggle_name: WIKI_ACCOUNT_HANDLING
+# .. toggle_implementation: DjangoSetting
+# .. toggle_default: False
+# .. toggle_description: We recommend you leave this as 'False' for an Open edX installation
+#   to get the proper behavior for register, login and logout. For the original docs see:
+#   https://github.com/edx/django-wiki/blob/edx_release/wiki/conf/settings.py
+# .. toggle_warnings: Turn to False only if you have your own account handling
```

Comment:
Please drop this as noted here: https://github.com/edx/edx-platform/pull/26234#discussion_r566567758

Reply:
@NeOneSoft: Thank you again. Once you drop this line I can merge.